homeassistant/components/zwave_js/sensor.py

```diff
@@ -271,6 +275,133 @@
     ),
 }
 
+# Controller statistics descriptions
+ENTITY_DESCRIPTION_CONTROLLER_STATISTICS_LIST = [
+    SensorEntityDescription(
```

Comment:
These should all be disabled by default.

Reply:
they are in the class definition, but will add to the ED